lua/lspconfig/server_configurations/tvm_ffi_navigator.lua

```diff
@@ -0,0 +1,25 @@
+local util = require 'lspconfig.util'
+
+return {
+  default_config = {
+    cmd = { 'python', '-m', 'ffi_navigator.langserver' },
+    filetypes = { 'python', 'cpp' },
+    root_dir = function(fname)
+      local root_files = {
+        'pyproject.toml',
+        '.git',
+      }
+      return util.root_pattern(unpack(root_files))(fname) or util.find_git_ancestor(fname)
+    end,
```

Comment:
```suggestion
    root_dir = return util.root_pattern('pyproject.toml', '.git')
```

Reply:
Yes, makes sense, thanks. I've made the changes